test/integration/taxon_browsing_test.rb

```diff
@@ -375,13 +362,6 @@ def and_i_can_see_content_tagged_to_the_taxon
   alias_method :and_i_can_see_content_tagged_to_the_taxon_and_the_associate,
     :and_i_can_see_content_tagged_to_the_taxon
 
-  def and_the_content_tagged_to_the_grandfather_taxon_has_tracking_attributes
```

Comment:
Mainly because the new logic is more complicated and I thought the integration test was not the right place to test it exhaustively. So now instead of testing if each tag has the right attributes (the old way and would now be too complicated), the new integration test only tests one specific case. The rest of the logic is tested in the taxon_presenter_test.